tests/queries/get-return-type.test.ts

```diff
@@ -0,0 +1,62 @@
+import testObject from '../../src/queries';
+import { transformToAst } from '../helpers';
+
+const testFunctionName = "getReturnType";
```

Comment:
You can create a constans.ts or strings.json file in which to put read-only(const) variable

Reply:
Great idea :) But in tests I need explicit name of function to see and use only here :)